misc/dbt-materialize/tests/adapter/test_deploy.py

```diff
@@ -336,35 +334,6 @@ def test_deployment_run(self, project):
         run_dbt(["run-operation", "deploy_cleanup"])
 
 
-class TestSinkFail:
```

Comment:
This test never worked as intended, it was failing with an error for missing kafka connection, rather than an error that sinks are not supported,